apis/v1alpha1/rhmi_types.go

```diff
@@ -37,9 +37,12 @@ var (
 	PhaseCreatingComponents     StatusPhase = "creating components"
 	PhaseAwaitingComponents     StatusPhase = "awaiting components"
 
-	PhaseInProgress StatusPhase = "in progress"
-	PhaseCompleted  StatusPhase = "completed"
-	PhaseFailed     StatusPhase = "failed"
+	PhaseInProgress          StatusPhase = "in progress"
+	PhaseCompleted           StatusPhase = "completed"
+	PhaseFailed              StatusPhase = "failed"
+	PhaseUninstallInProgress StatusPhase = "uninstall - in progress"
+	PhaseUninstallCompleted  StatusPhase = "uninstall - completed"
+	PhaseUninstallFailed     StatusPhase = "uninstall - failed"
```

Comment:
I'm interested in thoughts on the addition of these phases. I think it will force changes to products other than monitoring at the moment.

Reply:
I think adding these stages is a good idea but if the time required to fix changes to the other products is substantial (and the phases aren't required for the AMO uninstall) then it might be best to push it to the next sprint. If it's a simple fix though I think they're a useful addition.